db/schema.yml

```diff
@@ -710,6 +710,15 @@ container_images:
 - last_scan_attempt_on
 - digest
 - registered_on
+- architecture
+- author
+- command
+- entrypoint
+- docker_version
+- environment_varialbes
```

Comment:
The order here is different than the migration. They both need to match for the replication tests to pass. It should be changed to this order:
```
- exposed_ports
- environment_varialbes
- size
```